src/layout/LayoutComponent.tsx

```diff
@@ -190,25 +190,23 @@ export abstract class FormComponent<Type extends ComponentTypes>
     if (!node.item.required) {
       return [];
     }
+    const { langAsString } = langTools;
 
     const formDataToValidate = { ...formData, ...overrideFormData };
     const validationObjects: IValidationObject[] = [];
 
     const bindings = Object.entries(node.item.dataModelBindings ?? {});
     for (const [bindingKey, field] of bindings) {
       const data = formDataToValidate[field];
+      const textResourceBindings = node.item.textResourceBindings as ITextResourceBindings;
 
       if (!data?.length) {
-        const fieldName = getFieldName(node.item.textResourceBindings as ITextResourceBindings, langTools, bindingKey);
-
-        validationObjects.push(
-          buildValidationObject(
-            node,
-            'errors',
-            langTools.langAsString('form_filler.error_required', [fieldName]),
-            bindingKey,
-          ),
-        );
+        const fieldName = getFieldName(textResourceBindings, langTools, bindingKey);
+        const errorMessage = textResourceBindings?.requiredValidation
+          ? langAsString(textResourceBindings?.requiredValidation)
+          : langAsString('form_filler.error_required', [fieldName]);
```

Comment:
Is it an idea to add the `[fieldName]` parameter to the custom required validation text as well? If that works it is easier to reuse text resources if you want to use the same wording for different fields.

Reply:
Good idea! It seems to work well 😁